### ISSUE TITLE
Collaborator unable to upload phenotips

### ISSUE THREAD

**Issue by katie-larkin** [macarthur-lab/seqr#951]
Jennifer from the Gleeson lab is attempting to enter phenotips information for Gleeson Exome family 2231 https://seqr.broadinstitute.org/project/R0285_cmg_gleeson_exomes/family_page/F028246_2231 . According to her, "There is no error message.  It says it saved the pedigree but then when I opened the page to review it, it was blank."

Here is a screenshot she sent:
![image](https://user-images.githubusercontent.com/43212980/61246559-1897c300-a71d-11e9-8591-dc9993ec5c12.png)

Jaime's thought was that this may be due to a change in custom pedigree upload since their team's last submission, do you have any advice on how to help or further troubleshoot?

**Comment by hanars** [comment]
Can you find out exactly what steps they are doing to get that error?

**Comment by katie-larkin** [comment]
They got it working when I asked for step-by-step. Guessing it was due to user error.

**Comment by katie-larkin** [comment]
Sorry to reopen this but they just wrote back following up with the same issue.

It looks like they are able to view the phenotype entry from phenotips after they enter it, but the pedigree pdf they upload through phenotips is not appearing with the family. Does the update to pedigree uploads prevent phenotips pedigrees from saving? If so, I'll let them know the new way to do it and that we've phased out their current method.

**Comment by hanars** [comment]
The pedigree PDF in phenotips has never been used in seqr, anywhere. That said, they should be able to directly upload pedigrees to the family if that something they want to do

**Comment by katie-larkin** [comment]
Re-opening with another follow up from collaborator last Wednesday. Sam successfully loaded 6818 on Friday but collab still got error over the weekend.

"When tried to edit the pedigree files in some of the families in phenotips an error message was seen.  I listed the Gleeson family ID number where this happened.  
4983, 7240, 7247,7255, 7257, 7274, 7388, 7569, 7620, 6729,6811, 6818, 6819, 6821, 6856.
We finished putting all the info into the other families."

![image](https://user-images.githubusercontent.com/43212980/64560941-8cb6a780-d317-11e9-9821-02f3520f3f40.png)
